### PR TITLE
Docs multires fix

### PR DESCRIPTION
**Pull request by alexcg1**
Clean up, adding missing commands

### CODE REVIEW

**Review by maximilianwerk — APPROVED** [review]

File: multires-lyrics-search/README.md [hunk 1]
```diff
@@ -19,7 +19,14 @@
...
+
+We have included 1000 lyrics as toy data in [`toy-data`](toy-data).
+This data is ready to use with this example.
+
+## [Optional] Download full lyrics dataset
+
+If you want to use the full dataset, please run:
 
 ```bash
 pip install kaggle
```
Comment (JoanFM) [inline multires-lyrics-search/README.md:32]: it is important to say that they need to have the kaggle API key. I put a notice like this in cxmodal search example linking to kaggle info on how-to download one

Comment (maximilianwerk) [reply multires-lyrics-search/README.md:32]: Fair point. I'll add it.

Comment (alexcg1) [reply multires-lyrics-search/README.md:32]: So, if you don't download the full data it just uses the toy data automatically? Don't have to set any environment variables or fiddle with app.py?

Comment (maximilianwerk) [reply multires-lyrics-search/README.md:32]: :+1: I thought this would be a bit more useful for users ;)

**Pull request merged**